File: EaiGateway/src/main/java/com/yung/cho/eaigateway/config/ServiceRouteConfig.java
package com.yung.cho.eaigateway.config;

// 리우팅 정보 vo
public record ServiceRouteConfig (
        String uri,     // 타겟지 url
        long timeoutMs,     // 타겟지 타임아웃 ms
        float failureRateThreshold,     // 지난 10초간 실패율
        int concurrentSessions      // 동접 세션수
) {}

File: EaiGateway/src/main/kotlin/com/yung/cho/eaigateway/config/ServiceRouteConfig.kt
package com.yung.cho.eaigateway.config

/**
 * 라우팅 정보 vo
 */
data class ServiceRouteConfig(
    val uri: String,                // 타겟지 url
    val timeoutMs: Long,            // 타겟지 타임아웃 ms
    val failureRateThreshold: Float, // 지난 10초간 실패율
    val concurrentSessions: Int     // 동접 세션수
)
